sunpy/map/mapbase.py

```diff
@@ -163,10 +163,31 @@ class GenericMap(NDData):
         this class will not process the WCS information. This will be fixed.
         Also the rotation_matrix does not work if the CDELT1 and CDELT2
         keywords are exactly equal.
+        Also, if a file with more than two dimensions is loaded into the class,
+        only the first two dimensions (NAXIS1, NAXIS2) will be loaded, and the
+        rest will be discarded.
     """
 
     def __init__(self, data, header, **kwargs):
 
+
+        # If the data has more than two dimensions, the first dimensions
+        # (NAXIS1, NAXIS2) are used and the rest are discarded.
+        N_DIM = len(data.shape)
```

Comment:
The number of dimensions of the data is a property of numpy array, i.e. ndim = data.ndim. The definition of N_DIM should be replaced with ndim = data.ndim .